modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/SpringCodegen.java

```diff
@@ -546,26 +554,26 @@ public void processOpts() {
         if (isNotEmpty(responseWrapper)) {
             additionalProperties.put("jdk8-default-interface", false);
             switch (responseWrapper) {
-            case "Future":
-            case "Callable":
-            case "CompletableFuture":
-                additionalProperties.put(RESPONSE_WRAPPER, "java.util.concurrent." + responseWrapper);
-                break;
-            case "ListenableFuture":
-                additionalProperties.put(RESPONSE_WRAPPER, "org.springframework.util.concurrent.ListenableFuture");
-                break;
-            case "DeferredResult":
-                additionalProperties.put(RESPONSE_WRAPPER,
-                        "org.springframework.web.context.request.async.DeferredResult");
-                break;
-            case "RxObservable":
-                additionalProperties.put(RESPONSE_WRAPPER, "rx.Observable");
-                break;
-            case "RxSingle":
-                additionalProperties.put(RESPONSE_WRAPPER, "rx.Single");
-                break;
-            default:
-                break;
+                case "Future":
```

Comment:
Your IDE seems to have added a lot of auto-formatting - you may want to disable that and/or revert that. Makes it hard to review

Reply:
I removed them.